build/build.js

```diff
@@ -14,7 +14,6 @@
 	name: 'app',
 	out: '../js/mail.min.js',
 	insertRequire: [
-		'app',
-		'notification'
```

Comment:
cc @skjnldsv: I fixed another occurrence of that bloody module. I knew it was loaded twice, but I couldn't find it when I fix the other one…